indico/queries/model_groups/model_groups.py

```diff
@@ -236,3 +236,104 @@ def requests(self):
         yield _ModelGroupPredict(
             model_id=self.model_id, data=self.data, predict_options=self.predict_options
         )
+
+
+class UpdateModelGroupSettings(GraphQLRequest):
+    """
+    Updates an existing model group component in the platform.
+
+    Args:
+        model_group_id (int): the id of the model group to update settings
+        model_training_options (dict, optional): model training options to use when training model. Defaults to None
```

Comment:
Do we want to document the available model training options? Or at least link to them?